pxteditor/workspace.ts

```diff
@@ -22,6 +22,37 @@ namespace pxt.workspace {
         version: Version;
     }
 
+    export interface HistoryFile {
+        entries: HistoryEntry[];
+    }
+
+    export interface HistoryEntry {
+        timestamp: number;
```

Comment:
Maybe an editor version stamp here, or we could push a HistoryEntry whenever we load in a new version I suppose. (In theory we could scrape from pxt.json but that's messy / a little inconsistent)